package/main/daml/Daml.Finance.Claims/daml.yaml

```diff
@@ -11,10 +11,14 @@ dependencies:
 data-dependencies:
   - .lib/daml-finance/ContingentClaims.Core/0.1.1/contingent-claims-core-0.1.1.dar
   - .lib/daml-finance/ContingentClaims.Lifecycle/0.1.1/contingent-claims-lifecycle-0.1.1.dar
+  - .lib/daml-finance/Daml.Finance.Data/0.1.8/daml-finance-data-0.1.8.dar
```

Comment:
I think it would be nice to get rid of this cross-dependency.

This would mean moving `getHolidayCalendars` somewhere else and have the functions in here only take `HolidayCalendarData` as an input, which is defined in the `Interface` package

Reply:
Where would you like to move it? 
I had a quick look at moving it to Daml.Finance.Interface.Data, but that seems to mess up the build order.

Reply:
What about `Daml.Finance.Data`? 

We could then pass into `Dam.Finance.Claims` a function parameter with signature `Calendar Id -> Update HolidayCalendarData`. This way the two packages would be decoupled

Reply:
That would mean moving the dependency on `Daml.Finance.Data` to all the bond/swap instruments instead, since they call `rollPaymentSchedule` and would have to pass on `getHolidayCalendars` as a function parameter?

Reply:
That's right

Reply:
Maybe better merging this PR as-is, and possibly look at that separately?

Reply:
Opened up a separate issue for this: https://github.com/digital-asset/daml-finance/issues/386